ksql-parser/src/main/java/io/confluent/ksql/util/DataSourceExtractor.java

```diff
@@ -106,64 +98,26 @@ public boolean isJoin() {
     return isJoin;
   }
 
-  private final class Visitor extends SqlBaseBaseVisitor<AstNode> {
-
+  private final class Visitor2 extends DefaultTraversalVisitor<Void, Void> {
```

Comment:
Why the class name change?

Reply:
Ah that's a holdover from my changes. As I was evolving this there was a point where I had both visitors in this class. Will revert.